.changeset/eleven-roses-fry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
+---
```

Comment:
Lets bump the claims module as well

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/8090/commits/5f64408868d2aa169cc0966d62d7f8c1ad48f098